kivy/uix/bubble.py

```diff
@@ -235,6 +245,33 @@ def clear_widgets(self, **kwargs):
         else:
             content.clear_widgets()
 
+    def hide(self, *largs, **kwargs):
+        '''Close the Bubble if it is open.
+        When the view is dismissed, it will be faded out before being
+        removed from the parent. If you don't want animation, use::
+
+            view.dismiss(animation=False)
+
```

Comment:
this is a nice change, but i think instead of passing animation=False here, i would prefer the app to have an animation ObjectProperty, defaulting to the current animation you put, and with allownone=True, so one can disable it easily, or replace it with another animation.
also add the versionadded info.